addons/website_forum/views/website_forum.xml

```diff
@@ -441,30 +464,56 @@
                     <a t-attf-href="/forum/#{slug(forum)}">
-                        <div t-attf-class="oe_img_bg rounded shadow-sm py-4 px-5 o_wforum_forum_card_bg flex-shrink-0 h-100 o_not_editable {not has_desc and 'h-100'}"
-                            t-attf-style="#{forum.image_256 and ('background-image: url(%s);' % website.image_url(forum, 'image_256'))} background-position: center;"/>
+                        <div t-attf-class="oe_img_bg rounded shadow-sm o_wforum_forum_card_bg flex-shrink-0 h-100 o_editable #{not forum.image_1920 and not editable and 'opacity-0'}"
+                        t-field="forum.image_1920" t-options="{'widget': 'image'}" />
                     </a>
                 </div>
-                <div t-attf-class="mt-2 #{has_desc and 'col-md-8 mt-md-0'}">
+                <div class="mt-2 col-md-8 mt-md-0 d-flex flex-column">
                     <a t-attf-href="/forum/#{slug(forum)}" class="text-reset" t-att-title="forum.name">
-                        <h3 t-attf-class="#{has_desc and 'h4' or 'h5 m-0'}" t-field="forum.name"/>
+                        <h3 class="h4" t-field="forum.name"/>
                     </a>
-                    <p t-attf-class="m-0 #{has_desc and not no_description_exist and 'text-muted' or not has_desc and 'css_non_editable_mode_hidden'}"
+                    <p class="m-0 flex-grow-1"
                     placeholder="Description"
-                    t-field="forum.description"/>
+                    t-field="forum.teaser"/>
+                    <div t-if="opt_post_count or opt_last_post" class="row">
+                        <div class="col-md-8 pr-md-0 pt-3">
+                            <div t-if="opt_last_post and forum.post_ids">Last Post : <a t-attf-href="/forum/#{slug(forum)}/question/#{slug(forum.last_post_id)}"><t t-esc="forum.last_post_id.name"/></a></div>
+                        </div>
+                        <div class="col-md-4 pr-md-3 pt-3">
+                            <div t-if="opt_post_count">Posts : <strong><t t-esc="forum.total_posts"/></strong></div>
+                        </div>
+                    </div>
                 </div>
             </div>
         </div>
     </t>
 </template>
 
+<!-- (Options) Forum : List View
+    Display forums as a list  -->
+<template name="List View" id="website_forum.opt_list_view" inherit_id="website_forum.forum_all" active="False" customize_show="True"/>
+
+<!-- (Options) Forum : Show Post Count
+    Show the number of post a forum has  -->
+<template name="Show Post Count" id="website_forum.opt_post_count" inherit_id="website_forum.forum_all" active="False" customize_show="True"/>
+
+<!-- (Options) Forum : Show Last Post
+    Show the title of the latest post in each forum  -->
+<template name="Show Last Post" id="website_forum.opt_last_post" inherit_id="website_forum.forum_all" active="False" customize_show="True"/>
```

Comment:
Can't you use xpath for the customization for those 3? Instead of making a hack for no reason?

Reply:
+1